Exercises-03/exercise-01.js

```diff
@@ -0,0 +1,17 @@
+function flattenArray(arr) {
+    let flattenedArray = [];
+    if(!Array.isArray(arr)) return 0
```

Comment:
Here the function is returning 0 which I guess indicates the that the level of an array receive (No an array apparently).
But I would expect this function to return me a flattened array due to its function name and also in the below code there is return statement which returns a flattened array.


The point is a function should alway return the same data in terms of data type so that the person using it will not get unexpected value.

Always aim to getting rid of all surprises in the your code (we can discuss about this)